scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/JobResultView.java

```diff
@@ -60,6 +60,8 @@
 
     private static final String NO_JOB_SELECED = "No job selected.";
 
+    private static final String NO_ITEMS_TO_SHOW = "<b>Result List: no items to show.</b>";
```

Comment:
why bold?

Reply:
All titles were bold when I came. So I keep it bold.

Reply:
![image](https://user-images.githubusercontent.com/5878565/59209408-8e06f580-8bab-11e9-9806-5344df8df003.png)
it's not bold here

Reply:
it is inside ListGrid. it is another thing